feat(replay): Add information about adding CSP entry for Replay WebWorkers (#6269)
getsentry/sentry-docs: src/platforms/javascript/common/session-replay/index.mdx

<PlatformContent includePath="session-replay/setup" />

### Content Security Policy (CSP)

> Session Replay uses a WebWorker to perform work off the main UI thread as to not degrade performance of your application. Ensure that workers can be loaded from our CDN by adding this entry:
> 

Reviewer: ```suggestion
Session Replay uses a WebWorker to perform work off the main UI thread so as not to degrade the performance of your application. Add the below entry to make sure that workers can be loaded from our CDN:

```

<PlatformContent includePath="session-replay/setup" />

### Content Security Policy (CSP)

> Session Replay uses a WebWorker to perform work off the main UI thread so as not to degrade the performance of your application. Add the below entry to make sure that workers can be loaded from our CDN:

Author: Unfortunately, we can't load cross origin web workers :(

```suggestion
Session Replay uses a WebWorker to perform work (e.g. compression) off the main UI thread so as not to degrade the performance of your application. Add the below entry to make sure that workers can be loaded: 
```

```

> worker-src https://browser.sentry-cdn.com

Author: ```suggestion
worker-src 'self' blob:
```

```

> 

Author: ```suggestion

Older versions of Safari do not support `worker-src`, you will need to additionally add an entry for `child-src`:

child-src 'self' blob:
```

<PlatformContent includePath="session-replay/setup" />

### Content Security Policy (CSP)
Session Replay uses a WebWorker to perform work (e.g. compression) off the main UI thread so as not to degrade the performance of your application. Add the below entry to make sure that workers can be loaded: 

```
worker-src 'self' blob:
```

Older versions of Safari do not support `worker-src`, you will need to additionally add an entry for `child-src`:

> child-src 'self' blob:

Author: ```suggestion
```
child-src 'self' blob:
```

```

<PlatformContent includePath="session-replay/setup" />

### Content Security Policy (CSP)
Session Replay uses a WebWorker to perform work (e.g. compression) off the main UI thread so as not to degrade the performance of your application. Add the below entry to make sure that workers can be loaded: 

```
worker-src 'self' blob:
```

> Older versions of Safari do not support `worker-src`, you will need to additionally add an entry for `child-src`:

Reviewer 2: Do we know what versions exactly? As a reader I'm confused: Well I have users on Safari so do I need to add this too?

> Older versions of Safari do not support `worker-src`, you will need to additionally add an entry for `child-src`:

Author: ```suggestion
[Safari versions <= 15.4](https://caniuse.com/?search=worker-src) do not support `worker-src`, you will need to additionally add an entry for `child-src`:
```
